plugins/funind/recdef.ml

```diff
@@ -78,8 +78,10 @@ let def_of_const t =
 
 let type_of_const t =
    match (kind_of_term t) with
-    Const sp -> Typeops.type_of_constant (Global.env()) sp
-    |_ -> assert false
+   | Const sp ->
+      (* FIXME discarding universe constraints *)
```

Comment:
The FIXME should stay if type_of_const can be called on the body of a universe polymorphic function, then it should take an env to do type_of_constant_in in. Probably the FIXME should hence change